tests/credential-status/search.no-auth.spec.ts

```diff
@@ -0,0 +1,97 @@
+import * as fs from 'fs';
+import {
+    TESTNET_DID,
+    INVALID_DID,
+    PAYLOADS_PATH,
+    NOT_EXISTENT_TESTNET_DID,
+    NOT_EXISTENT_STATUS_LIST_NAME
+} from '../constants';
+import { test, expect } from '@playwright/test';
+import { StatusCodes } from 'http-status-codes';
+
+test('[Positive] It can search credential-status with existent DID, statusListName, and statusPurpose=revocation', async ({ request }) => {
+    const response = await request.get(
+        '/credential-status/search?' +
+        `did=${TESTNET_DID}&` +
+        'statusPurpose=revocation&' +
+        'statusListName=cheqd-employee-credentials'
+    );
+    expect(response.status()).toBe(StatusCodes.OK);
+
+    const body = await response.json();
+    const expected = JSON.parse(
```

Comment:
The same situation here as before. Cause we are using the real testnet DIDs in case if this DID be changed we will get false-negative scenario here.